pkg/local_object_storage/shard/delete.go

```diff
@@ -63,7 +64,9 @@ func (s *Shard) Delete(prm DeletePrm) (*DeleteRes, error) {
 
 	err := meta.Delete(s.metaBase, prm.addr...)
 	if err != nil {
-		return nil, err // stop on metabase error ?
+		if mode&ModeDegraded == 0 { // Skip metabase errors in degraded mode.
```

Comment:
why have we even tried to `Delete` anything in the `degraded moe?`

Reply:
fixed